lib/tasks/version.rake

```diff
@@ -36,4 +36,31 @@ namespace :version do
       File.write(spec_file, spec)
     end
   end
+
+  desc "Bump package version and create a simple class for acessing the version from sources"
+  task wbump: :bump do
+    puts "Generating helper class"
+
+    Dir.glob("**/lib/*") do |filename|
+      next if !File.directory?(filename)
```

Comment:
Name `filename` is a bit misleading here, it actually contains a *directory* name. Rather use `path` or `dirname` in this case.